modules/doc/content/newsletter/index.md

```diff
@@ -6,7 +6,7 @@ monthly to the [MOOSE users](contact_us.md) as well as provided below.
 
 ## 2019
 
-- [February, 2019](2019_03.md)
+- [March, 2019](2019_03.md)
```

Comment:
I don't want to list March yet, when we hit March I will add the link and send out the announcement. There should just be Jan. and Feb. listed.